Generator/OdmGenerator.php

```diff
@@ -45,24 +45,27 @@ public function generate(OutputInterface $output, BundleMetadata $bundleMetadata
     /**
      * @return string
      */
-    public function getDocumentTemplate()
+    public function getDocumentTemplate(): string
     {
         return $this->documentTemplate;
     }
 
     /**
      * @return string
      */
-    public function getDocumentRepositoryTemplate()
+    public function getDocumentRepositoryTemplate(): string
     {
         return $this->documentRepositoryTemplate;
     }
 
     /**
      * @param OutputInterface $output
      * @param BundleMetadata  $bundleMetadata
+     * @param string
```

Comment:
:thinking:

Reply:
I knew something was off when I fixed that for the second time :P